hot_peaks_table.py

```diff
@@ -111,8 +112,54 @@ def bool_variant_df(df):
 
 # ... (similar explanations for other functions)
 
-
-
+# some rows have a few genes in 'DSDgenes_1.5mb' and this function splits them to several rows
+def handle_multi_gene_rows(df):
+    # Create a copy of the DataFrame to work with
+    new_df = df.copy()
+    df.reset_index(inplace=True)
+    # Split values in 'DSDgenes_1.5mb' column and create new rows
+    new_rows = []
+    for index, row in new_df.iterrows():
```

Comment:
```suggestion
def help_func(x, gene_list):
      new_df = pd.concat([x]* len(gene_list)) # might need to transpose [df.T]
      new_df['DSDgenes_1.5mb'] = pd.Series(gene_list)
      return new_df

    no_forloop= new_df['DSDgenes_1.5mb'].str.split(',').apply(lambda x: help_func(new_df.iloc[x.index,:], x))
   
```

Reply:
Apply is kind of looping BUT it is more effciant and make the code more readable. 
I did not checked my code. please make sure it works :)